index.html

```diff
@@ -9035,6 +9041,74 @@ <h2>Screen capture</h2>
  and a <a>command</a> <a>Take Element Screenshot</a>
  for doing the same with the visible region of an <a>element</a>’s
  <a>bounding rectangle</a> after it has been <a>scrolled into view</a>.
+ It also provides a <a>Take Fullpage Screenshot</a> <a>command</a>
+ to optionally capture the <a>fullpage region</a>.
+
+<p>An <a>element</a>’s <dfn data-lt="fullpage region">fullpage region</dfn>
+ is the rendered rectangle that encompasses the maximal scrollable width
+ and height,
+ it can be calculated this way:
+<ol>
+ <li><p>Let <var>x</var> be 0
+
+ <li><p>Let <var>y</var> be 0
+
+ <li><p>Let <var>width</var> be the value
+  returned by
+  <p><a>max</a>(document.body.scrollWidth, document.documentElement.scrollWidth,
+                document.body.offsetWidth, document.documentElement.offsetWidth,
+                document.body.clientWidth, document.documentElement.clientWidth)
+
+ <li><p>Let <var>height</var> be the value
+  returned by
+  <p><a>max</a>(document.body.scrollHeight, document.documentElement.scrollHeight,
+                document.body.offsetHeight, document.documentElement.offsetHeight,
+                document.body.clientHeight, document.documentElement.clientHeight)
+
+  <li><p>Return {<var>x</var>, <var>y</var>, <var>width</var>, <var>height</var>}.
+</ol>
+
+<p>In order to <dfn>draw a bounding box from the fullpage region</dfn>,
```

Comment:
Could this just reuse the "draw a bounding box from the framebuffer" steps? It seems to be mostly duplicated.